Gem/Code/CMakeLists.txt

```diff
@@ -52,6 +54,7 @@ ly_add_target(
     FILES_CMAKE
         multiplayersample_autogen_files.cmake
         multiplayersample_files.cmake
+        multiplayersample_dedicated_server_files.cmake
```

Comment:
nit: it may be better to tie this to gamelift as it made me double take that we may have been missing something